### PR TITLE
RN 39113 + 38710

### PR DESCRIPTION
**Pull request by MariekeGO**

### CODE REVIEW

**Comment by MariekeGO** [comment]
@JordyGit, is a restart of the DxMs necessary to apply these changes?

**Comment by JordyGit** [comment]
> @JordyGit, is a restart of the DxMs necessary to apply these changes?

Yes each specific DxM where you change a setting for needs to be restarted.

File: user-guide/Cloud_Platform/CloudAdminApp/Controlling_remote_access.md [hunk 1]
```diff
@@ -23,3 +23,4 @@ If you have the Owner or Admin role on dataminer.services, you can enable or dis
 >
 > - When you change the *Remote Access* setting from *On* to *Off*, it might take a few minutes before access is blocked.
 > - Users will only be able to use the remote access feature if they have been given access to the DMS on dataminer.services. See [Controlling user access to dataminer.services features](xref:Giving_users_access_to_cloud_features). They also need to have the [General > DataMiner web apps](xref:DataMiner_user_permissions#general--dataminer-web-apps--dataminer-cube-mobile-access) user permission, as well as any other user permissions required to access specific apps.
+> - The Remote Access feature can also be disabled locally on a specific server. See [Disabling Remote Access and Live Sharing locally](xref:Disabling_Remote_Access_and_Live_Sharing)
```
Comment (JordyGit) [inline user-guide/Cloud_Platform/CloudAdminApp/Controlling_remote_access.md:26]:
```suggestion
> - The Remote Access feature can also be disabled locally on a specific server. See [Disabling Remote Access and Live Sharing locally](xref:Disabling_Remote_Access_and_Live_Sharing).
```

File: user-guide/Cloud_Platform/Advanced_config/Disabling_Remote_Access_and_Live_Sharing.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+uid: Disabling_Remote_Access_and_Live_Sharing
+---
+
+# Disabling Remote Access and Live Sharing locally
+
+From version 2.13.8 of the [CloudGateway](xref:DataMinerExtensionModules#cloudgateway) DxM onwards<!-- RN 39113 -->, it is possible to disable the Remote Access and Live Sharing features for a particular server through the app settings of the DxM:
+
+1. On the server where you want to disable these features, go to the folder `C:\Program Files\Skyline Communications\DataMiner CloudGateway`.
```
Comment (JordyGit) [inline user-guide/Cloud_Platform/Advanced_config/Disabling_Remote_Access_and_Live_Sharing.md:9]: maybe here 'on each server' to be correct & aligned with the restart step

**Review by JordyGit — APPROVED** [review]

File: user-guide/Cloud_Platform/Advanced_config/Enabling_kernel_response_buffering.md
[the remark this answers is not in the document]
Comment (bertbsky) [reply user-guide/Cloud_Platform/Advanced_config/Enabling_kernel_response_buffering.md:1]: Looks OK to me.
Note: we expect to make this setting the default in the future (when making gRPC the default communication type), so at that time this documentation can be inverted (how to disable it again incase of memory problems).

**Pull request merged**